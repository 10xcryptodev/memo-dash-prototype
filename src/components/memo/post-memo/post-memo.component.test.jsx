import React from 'react'
import { shallow } from 'enzyme'
import PostMemoComponent from './post-memo.component'

describe('<PostMemoComponent />', () => {
  let wrapper

  beforeEach(() => {
    const div = document.createElement('div')
    document.body.appendChild(div)
  })

  it('renders without crashing', () => {
    wrapper = shallow(
      <PostMemoComponent
        values={{ message: 'message' }}
        errors={{ message: 'errorMessage' }}
        touched={{ message: 'touchedMessage' }}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
