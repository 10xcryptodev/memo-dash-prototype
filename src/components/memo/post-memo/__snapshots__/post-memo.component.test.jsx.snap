// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostMemoComponent /> renders without crashing 1`] = `
<Segment
  className="post-memo"
  color="blue"
>
  <Form
    as="form"
    reply={true}
  >
    <FormTextArea
      as={[Function]}
      autoHeight={true}
      className=" error"
      control={[Function]}
      name="message"
      placeholder="Your Message"
      style={
        Object {
          "minHeight": 20,
        }
      }
      value="message"
    />
    <Button
      as="button"
      content="Post Memo"
      disabled={true}
      icon="edit"
      labelPosition="left"
      primary={true}
      role="button"
      type="submit"
    />
  </Form>
</Segment>
`;
